src/main/java/de/dennisguse/opentracks/util/IntentDashboardUtils.java

```diff
@@ -58,6 +58,20 @@ private IntentDashboardUtils() {
      * @param trackIds the track ids
      */
     public static void startDashboard(Context context, boolean isRecording, Track.Id... trackIds) {
+        startDashboard(context, isRecording, null, null, trackIds);
+    }
+
+    /**
+     * Send intent to show tracks on a map (needs an another app) as resource URIs. By providing a
+     * targetPackage and targetClass an explicit intent can be sent, thus bypassing the need for the
+     * user to select an app.
+     *
+     * @param context  the context
```

Comment:
not necessary as the variable names already tell the story.

Reply:
Fixed